dictionaries/CallMap_73_delta.php

```diff
@@ -26,23 +26,23 @@
     'JsonException::getLine' => ['int'],
     'JsonException::getMessage' => ['string'],
     'JsonException::getPrevious' => ['?Throwable'],
-    'JsonException::getTrace' => ['list<array{file?:string,line?:int,function:string,class?:class-string,type?:\'::\'|\'->\',args?:array<mixed>}>'],
+    'JsonException::getTrace' => ['list<strict-array{file?:string,line?:int,function:string,class?:class-string,type?:\'::\'|\'->\',args?:array<mixed>}>'],
```

Comment:
should we have any param documented as strict-array in callmaps? There's no internal php that will complain about receiving too much things I think

Reply:
True, though imo strict arrays should still be used in return types